report/sections/abstract.tex

```diff
@@ -0,0 +1,5 @@
+\begin{abstract}
```

Comment:
That must be it, I was thinking about co-op reports. I followed your link and I see what you mean. Having it formatted the 2 column way is fine, though it might change the flow of the whole report, otherwise the way you had it before, with the abstract as a regular section is also okay.